src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        Random random = new Random(Color.values().length);
```

Comment:
The `Random` constructor should not take an argument. Instead, use `random.nextInt(Color.values().length)` to ensure the random index is within the bounds of the `Color` array.